packages/mdc-tabs/tab/mdc-tab.scss

```diff
@@ -32,7 +32,7 @@ $mdc-tab-with-icon-and-text-height: 72px;
 // postcss-bem-linter: define tab
 
 .mdc-tab {
-  @include mdc-typography(body2);
+  @include mdc-typography(button);
```

Comment:
What's the reason for using button here now?

Also, have you checked Pat's branch for the new Tab packages at all? They're going to need typography updates too, I'm sure...

Reply:
The latest guidance indicates Tab should use `button`.